examples/emoji_pet.py

```diff
@@ -0,0 +1,66 @@
+"""
+    emoji_pet.py
+    ~~~~~~~~
+
+    A reactive "pet" which changes face depending on certain inputs
+    editable by the user. Emotions are functions of an animation function.
```

Comment:
Perhaps, "Emotions are represented by animations that use the display.show function"..?